REVIEWING.md

```diff
@@ -0,0 +1,317 @@
+
+### Reviewing Changeset Bump Levels
+
+The following table provides a reference for what type of version bump is needed for each package. This is applied separately to each individual package, it does not matter what the scope of a change is in any other broader context.
+
+| Scope           | Current Package Version | Bump Level |
+| --------------- | ----------------------- | ---------- |
+| Breaking Change | `1.0` and above         | `major`    |
+| New Feature     | `1.0` and above         | `minor`    |
+| Fix             | `1.0` and above         | `patch`    |
+| Breaking Change | `0.x`                   | `minor`    |
+| New Feature     | `0.x`                   | `patch`    |
+| Fix             | `0.x`                   | `patch`    |
+
+The only situation where a package that is currently at `0.x` can have a `major` bump is if all owners and stakeholders of the package agree that the package is ready to be released as `1.0`.
+
+### Reviewing Changeset Content
+
+Each changeset should be written in a way that describes the impact of the change for the users of each package. The contents of the changesets will end up in the changelog of each package, for example [@backstage/core-plugin-api](./packages/core-plugin-api/CHANGELOG.md). The changelogs are intended to provide both a summary of the new features as well as guidance in the case of breaking changes or deprecations.
+
+Some things that changeset should NOT contain are:
+
+- Internal architecture details - these are generally not interesting to users, focus on the impact towards users of the package instead.
+- Information related to a different package.
+- A large amount of content, consider for example a separate migration guide instead, either in the package README or [./docs/](./docs/), and then link to that instead.
+- Documentation - changesets can describe new features, but it should not be relied on for documenting them. Documentation should either be placed in [TSDoc](https://tsdoc.org) comments, package README, or [./docs/](./docs/).
+
+### When is a changeset needed?
+
+In general our changeset feedback bot will take care of informing whether a changeset is needed or not, but there are some edge cases. Whether a changeset is needed depends mostly on what files have been changed, but sometimes also on the kind of change that has been made.
+
+Changes that do NOT need a new changeset:
+
+- Changes to any test, storybook, or other local development files, for example, `MyComponent.test.tsx`, `MyComponent.stories.tsx, `**mocks**/MyMock.ts`, `.eslintrc.js`, `setupTests.ts`, or `api-report.md`. Explained differently, it is only files that affect the published package that need changesets, such as source files and additional resources like `package.json`, `README.md`, `config.d.ts`, etc.
```

Comment:
Since the readme text is part of the published package info to npm, yeah actually it does!

Reply:
Even a build badge would need a changeset if you want to make sure that the package actually gets published. Otherwise the build badge might not start showing up in NPM. The other rule applies too though, for example if you remove a double newline from a markdown doc that doesn't really have an impact and doesn't need a changeset.